docs/en/trust.rst

```diff
@@ -602,28 +604,83 @@ The Trust Chains can also be verified offline, using one of the Trust Anchor's p
     The Wallet Attestation conveys all the required information pertaining to the instance, such as its public key and any other technical or administrative information, without any User's personal data.
 
 
-Relying Party Trust Evaluation
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Establishing Trust with Relying Parties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Relying Party is registered by a Trust Anchor or its Intermediate and obtains a Trust Mark to be included in its Entity Configuration. In its Entity Configuration the Relying Party publishes its specific metadata, including the supported signature and encryption algorithms and any other necessary information for the interoperability requirements.
+The Relying Party is registered by a Trust Anchor or its Intermediate and obtains a Trust Mark to be included in its Entity Configuration.
+In its Entity Configuration the Relying Party publishes its specific metadata, including the supported signature and encryption algorithms
+and any other necessary information for the interoperability requirements.
 
-Any requests for User attributes, such as PID or (Q)EAA, from the Relying Party to Wallet Instances are signed and SHOULD contain the verifiable Trust Chain regarding the Relying Party.
+Any requests for User attributes, such as PID or (Q)EAA, from the Relying Party to Wallet Instances are signed and SHOULD contain the verifiable
```

Comment:
```suggestion
Any requests for User attributes, such as PID or (Q)EAA, from the Relying Party to Wallet Instances are signed using one of the protocol related keys included in the metadata of the Relying Party.
```

Reply:
according to the work made so far in openid federation wallet arch draft, the cryptographic material will be definitively provided by .well-known/jwt-vc-issuer and this will be reflected also in openid_credential_issuer

since these changes are not supposed to be acquired in this release, I suggest to keep it generic for now and reduce the amount of changes we expect for the next releases

Reply:
I don't understand your comment. In the suggestion I referred to openid_relying_party metadata and not metadata of credential Issuers.

Reply:
unfortunately this applyies to both credential issuer and verifiers, since them both are jwt vc issuer

since this is a huge conversation still on the way, I suggest to relax our wording making it more flexible and applicable with different technical solutions

the end of this road will happen with the evolution of the openid4vci specs and the federation wallet arch (in course of approval)

for the time being, I would reduce the amount of fixed stuffs in multiple parts, to reduce the amount of changes we might have in the future